apps/code-examples/src/app/code-examples/validation/url-validation/control-validator/url-validation-demo.component.ts

```diff
@@ -21,6 +21,10 @@ export class UrlValidationDemoComponent implements OnInit {
 
   constructor(private formBuilder: FormBuilder) {}
 
+  public skyUrlValidationOptions: SkyUrlValidationOptions = {
+    rulesetVersion: 2,
+  };
```

Comment:
Is this property being used? I think it's meant to be put in the `FormControl`, below?
```
SkyValidators.url({
  rulesetVersion: 2,
})
```

Reply:
resolved in: [needs to use v2 ACTUALLY in code examples for control validator](https://github.com/blackbaud/skyux/pull/201/commits/94080c1c7d0d594eeb613b7d92272a34a1d5434a)